src/services/user-config-service.ts

```diff
@@ -28,8 +27,7 @@ const MAX_SERVICE_ID_COUNT = 100;
 export const updateRepoConfig = async (
 	subscription: Subscription,
 	repositoryId: number,
-	githubInstallationId: InstallationId,
-	metrics: { trigger: string, subTrigger?: string },
+	client: GitHubInstallationClient,
```

Comment:
The JSDoc params above this function won't match anymore. My suggestion: just delete them. It's not recommended to mix JSDoc and Typescript. Plus, TypeScript is pretty rock-solid. If these were added to help clarify what the params are because they're confusing I'd say we should be looking at renaming these.

Reply:
/nit can we use `gitHubInstallationClient: GitHubInstallationClient` as the variable name. We are slowly getting them all to that lol

Reply:
fixed.